features/end_to_end/et3_completed_response_form_available_to_download.feature

```diff
@@ -14,17 +14,17 @@ Feature: ET3 response to local office
   Scenario: PDF Format
     Given an employer responds to a claim
     When the completed Employment Tribunal response form is submitted
-    Then I can download the ET3 form and validate in PDF format
+#    Then I can download the ET3 form and validate in PDF format - Currently failing due to difficult error 4/01/2022 GoTo: https://tools.hmcts.net/jira/browse/RST-4945
```

Comment:
There is a normal way of marking something as needing doing that IDE's etc.. pick up - its called a todo tag and its written like this

```ruby
#TODO Currently failing due to difficult error (RST-4945)

```